nims-notes-things/src/Form.js

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import TextArea from "./TextArea";
+import Input from "./Input";
+import Button from "./Button";
+
+const Form = (props) => {
+  const [formData, setFormData] = useState({
+    name: "",
+    date: "",
+    note: "",
+  });
+
+  const [empty, setFormEmpty] = useState({
+    name: false,
+    date: false,
+    note: false,
+  });
+
+  const handleOnChange = (e, key) => {
+    const newForm = {
+      ...formData,
+      [key]: e.target.value,
+    };
+    setFormData(newForm);
+  };
+
+  const handleOnBlur = (value, key) => {
+    setFormEmpty((previousState) => {
+      return {
+        ...previousState,
+        [key]: value === "",
+      };
+    });
+  };
+
+  return (
+    <form>
```

Comment:
Not a massive deal, but have told to do this myself before. Instead of a pure html element maybe add a styled component in case you want to add more style in the future. Even if they are empty for now.

```
<StyledForm>
   <StyledDiv>
      // Rest of the code
   </StyledDiv>
</StyledForm>

const StyledForm = styled.form``;
const StyledDiv = styled.div``;
```

Reply:
You can call them whatever you want